scripts/update_VGA.sh

```diff
@@ -32,18 +32,10 @@ trap 'echo An error occurred in $0 at line $LINENO. Current working-dir: $PWD' E
 if [ -r /media/*/VBOXADDITIONS*/VBoxLinuxAdditions.run ]; then
   VGArun=/media/*/VBOXADDITIONS*/VBoxLinuxAdditions.run
 else
-  # get the VB version using a trick from Ben Thomas
```

Comment:
potentially leave the code in (either commented or just by putting the error_message+exit first) for future reference (although we can get it back from git of course)